### PR TITLE
Start working with first map and tileset

### PR DESCRIPTION
**Pull request by AedenSpear**

### CODE REVIEW

File: Incorpreal/player/Player.tscn [hunk 1]
```diff
@@ -113,37 +68,25 @@ tracks/0/keys = {
...
 [node name="player" type="Sprite" parent="Sprite"]
 texture = ExtResource( 1 )
 
 [node name="AnimationPlayer" type="AnimationPlayer" parent="."]
-
 anims/Die = SubResource( 2 )
 anims/Idle = SubResource( 3 )
 anims/Walking = SubResource( 4 )
 
+[node name="Camera2D" type="Camera2D" parent="."]
```
Comment (AedenSpear) [inline Incorpreal/player/Player.tscn:88]: Every change above this can be scrapped.

File: Incorpreal/player/Player.cs [hunk 2]
```diff
@@ -2,157 +2,157 @@
...
-          break;
-        }
-        default: {
-          break;
-        }
-      }
-    }
+	[Export]
+
+	public int moveSpeed = 125;
```
Comment (AedenSpear) [inline Incorpreal/player/Player.cs:7]: Everything other than movement speed can be ignored.

File: Incorpreal/Battle/Battle.cs [hunk 3]
```diff
@@ -4,8 +4,8 @@
 public class Battle : Node {
 
   //This is just for demo expamples to not get stuck on battle screen.
-    public void _on_Button_pressed() {
-        GetTree().ChangeScene("res://Level 1.tscn");
-    }
+	public void _on_Button_pressed() {
```
Comment (AedenSpear) [inline Incorpreal/Battle/Battle.cs:7]: Discard my changes here

**Comment by Luke-Ed** [comment]
I'd like someone else to double check this, I had to make some changes to the Level 1.tcsn file due to it not opening. Likely some fragments from your merge with main.

**Comment by MaverickEngan** [comment]
From what I can see it seems to be working properly.

**Pull request merged**